src/containers/ProjectComponentLoader.jsx

```diff
@@ -21,7 +20,6 @@ import { useProjectPersistence } from "../hooks/useProjectPersistence";
 const ProjectComponentLoader = (props) => {
   const loading = useSelector((state) => state.editor.loading);
   const { identifier } = useParams();
-  const embedded = props.embedded || false;
```

Comment:
Might be missing something, but why is this now redundant?

Reply:
> Might be missing something, but why is this now redundant?

The route that I removed seemed to be the only place where the `embedded` property for `ProjectComponentLoader` was set to `true`. The `embedded` property itself defaults to `false` and thus after the route was removed the value of the property must always be `false`. The only thing the `embedded` proeprty was used for was to call `useEmbeddedMode()`. However, that function is a no-op when `embedded` is `false`. So we can remove all of it. Does that make sense?

Reply:
In case it helps, there is still an `embedded` attribute on the web component: https://github.com/RaspberryPiFoundation/editor-ui/blob/340ab1192fb3358ad4614034208603948f8b95d6/src/web-component.js#L50 which is still used in `WebComponentLoader`.

Reply:
Ah I see, yes, that makes sense - I was worried about some of the embedded viewer stuff, but you're right, that's not using the `ProjectComponentLoader` so it should be fine 👍